kani-driver/src/call_cargo.rs

```diff
@@ -46,10 +46,39 @@ pub struct CargoOutputs {
 
 impl KaniSession {
     /// Create a new cargo library in the given path.
+    ///
+    /// Since we cannot create a new workspace with `cargo init --lib`, we create the dummy
+    /// crate manually. =( See <https://github.com/rust-lang/cargo/issues/8365>.
+    ///
+    /// Without setting up a new workspace, cargo init will modify the workspace where this is
```

Comment:
Could you add ~1 sentence to this command explaining how exactly it modifies the workspace (i.e., what bad modifications we're trying to prevent)?

Reply:
Alternatively, can you link to the Kani issue #3574 here as well, so that people can see the diff for themselves?

Reply:
sure